src/main/java/uk/gov/hmcts/reform/pip/account/management/controllers/AccountController.java

```diff
@@ -133,16 +134,32 @@ public ResponseEntity<String> deleteAccount(@PathVariable UUID userId) {
         return ResponseEntity.ok("User deleted");
     }
 
+    @ApiResponse(responseCode = OK_CODE, description = "User deleted")
+    @ApiResponse(responseCode = AUTH_ERROR_CODE, description = NOT_AUTHORIZED_MESSAGE)
+    @ApiResponse(responseCode = NOT_FOUND_ERROR_CODE, description = "User not found")
+    @ApiResponse(responseCode = FORBIDDEN_ERROR_CODE,
+        description = "User with ID %s is forbidden to remove user with ID %s")
+    @Operation(summary = "Delete a user by their id")
+    @DeleteMapping("/v2/{userId}")
+    @PreAuthorize("@authorisationService.userCanDeleteAccount(#userId, #adminUserId)")
+    public ResponseEntity<String> deleteAccountV2(@PathVariable UUID userId,
+                                                @RequestHeader("x-admin-id") UUID adminUserId) {
+        accountService.deleteAccount(userId);
+        return ResponseEntity.ok("User deleted");
+    }
+
     @ApiResponse(responseCode = OK_CODE, description = "String confirming update")
     @ApiResponse(responseCode = AUTH_ERROR_CODE, description = NOT_AUTHORIZED_MESSAGE)
     @ApiResponse(responseCode = NOT_FOUND_ERROR_CODE, description = "User not found")
-    @ApiResponse(responseCode = FORBIDDEN_ERROR_CODE, description = "User with id %s is unable to update user ID %s")
+    @ApiResponse(responseCode = FORBIDDEN_ERROR_CODE,
+        description = "User with ID %s is forbidden to update user with ID %s")
     @Operation(summary = "Update a users role by their id")
     @PutMapping("/update/{userId}/{role}")
+    @PreAuthorize("@authorisationService.userCanUpdateAccount(#userId, #adminUserId)")
     public ResponseEntity<String> updateAccountRoleById(@PathVariable UUID userId,
                                                         @PathVariable Roles role,
                                                         @RequestHeader(value = "x-admin-id", required = false)
```

Comment:
Can required now be true for this?

Reply:
Removed required=false to use the default true value